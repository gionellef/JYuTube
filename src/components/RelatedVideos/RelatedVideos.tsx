import * as React from 'react';

import { VideoPreview } from '../VideoPreview/VideoPreview';
import { NextUpVideo } from './NextUpVideo/NextUpVideo';

import './RelatedVideos.scss';

interface Props {
  className?: string;
  horizontal?: boolean,
};

const RelatedVideos : React.FC<Props> = (props) => {
  return (
    <React.Fragment>
      <div className='related-videos'>
        <NextUpVideo />
        <VideoPreview horizontal={true} />
        <VideoPreview horizontal={true} />
        <VideoPreview horizontal={true} />
        <VideoPreview horizontal={true} />
        <VideoPreview horizontal={true} />
      </div>
    </React.Fragment>
  );
}

export default RelatedVideos;