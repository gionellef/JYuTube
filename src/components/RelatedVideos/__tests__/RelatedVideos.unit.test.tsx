import * as React from 'react';

import { shallow } from 'enzyme';

import RelatedVideos from '../RelatedVideos';

test('RelatedVideos', () => {
  const wrapper = shallow (
    <RelatedVideos />
  );

  expect(wrapper).toMatchSnapshot();
});