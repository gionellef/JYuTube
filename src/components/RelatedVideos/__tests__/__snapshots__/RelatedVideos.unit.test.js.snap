// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RelatedVideos 1`] = `
<Fragment>
  <div
    className="related-videos"
  >
    <NextUpVideo />
    <VideoPreview
      horizontal={true}
    />
    <VideoPreview
      horizontal={true}
    />
    <VideoPreview
      horizontal={true}
    />
  </div>
</Fragment>
`;
